README.md

```diff
@@ -253,3 +253,14 @@ Alternatively, if you only care about the version you're currently running, you
 5. Run `twine upload dist/*` to deploy the distributions to PyPi
 
 You will need to be set up as a maintainer in order to do this.
+
+
+## Tracking
```

Comment:
We also need a mention of this in the ChangeLog.md

Reply:
Oh yeah, thanks for the reminder 👍